kong/plugins/url-redirect/schema.lua

```diff
@@ -0,0 +1,27 @@
+local typedefs = require "kong.db.schema.typedefs"
+local PLUGIN_NAME = "kong-plugin-redirect"
```

Comment:
`kong-plugin-redirect` is repo name, not plugin name

```suggestion
local PLUGIN_NAME = "kong-redirect"
```